pom.xml

```diff
@@ -1030,7 +1030,7 @@
       <dependency>
         <groupId>org.hsqldb</groupId>
         <artifactId>hsqldb</artifactId>
-        <version>2.4.1</version>
+        <version>2.7.1</version>
```

Comment:
This is a Java 11 (class file version 55.0) build so will cause failures when used with any Java version before 11.
Most of the CI is still running on Java 8

Reply:
> This is a Java 11 (class file version 55.0) build so will cause failures when used with any Java version before 11.

Sorry, I figured this out after having opened the PR.

Reply:
We can  merge once we switch main to Java 11... we'll ping you when it's done, so that you can rebase this PR.

Reply:
ok, thanks @aaime